services/ui-src/patches/@cmsgov+design-system+2.13.0.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/node_modules/@cmsgov/design-system/dist/scss/components/_Pagination.scss b/node_modules/@cmsgov/design-system/dist/scss/components/_Pagination.scss
+index 10da57c..bdf4b0d 100644
+--- a/node_modules/@cmsgov/design-system/dist/scss/components/_Pagination.scss
++++ b/node_modules/@cmsgov/design-system/dist/scss/components/_Pagination.scss
+@@ -9,7 +9,7 @@ $pagination-overflow-color: $color-gray;
+ $pagination-page-count-color: $color-gray;
+ 
+ .ds-c-pagination {
+-  align-items: end;
++  align-items: flex-end;
+   display: flex;
+   justify-content: space-between;
+   ul {
```

Comment:
i know that we did this to get rid of some console errors from the @cmsgov/design-system package. But after looking at the package it looks as though we should be able to just update it to the latest version (3.3.0) and then remove this file and the postinstall script.

Reply:
yea I agree